docs/source/api/python/dgl.rst

```diff
@@ -208,7 +208,7 @@ set at each iteration. ``prop_edges_YYY`` applies traversal algorithm ``YYY`` an
 Utilities
 -----------------------------------------------
 
-Other utilities for controlling randomness, saving and loading graphs, functions that applies
+Other utilities for controlling randomness, saving and loading graphs, turn on/off libxsmm, functions that applies
```

Comment:
To avoid future changes, use a more general term, e.g., `setting / getting runtime configurations`.

Reply:
Make sense, replace it with configuration description